### PR TITLE
Bundles for July 2021

### PR DESCRIPTION
**Pull request by tshoang**
- Based on Rodin 3.6.0
- Update CamilleX to 2.1.0
- Update Scenario Checker to 1.0.0
- Update ProB Support to 0.0.1
- Include RMF 0.13
- Include Sphinx 0.11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,71 +5,81 @@ group. Currently the folllowing bundles are included:
...
+  - EMF SDK 2.24.0.v20200917-1439
+  - GMF Tooling SDK 3.2.1.201409171321
+  - RMF ProR SDK 0.13.0.201509161042
+  - Sphinx SDK 0.11.0.201706140911
 
 Running *mvn clean verify* at the top level will build all
 bundles. Individual bundle can be built by running *mvn clean verify*
 within the corresponding folder.
 
+We recommend you to install Atelier-B provers (via Atelier-B Update
```
Comment (cfsnook) [inline README.md]: We recommend that you install

File: README.md [hunk 2]
```diff
@@ -5,71 +5,81 @@ group. Currently the folllowing bundles are included:
...
+Site) and Relevant filter (from the Rodin Plug-ins update site) to
```
Comment (cfsnook) [inline README.md]: Relevance

File: README.md [hunk 3]
```diff
@@ -5,71 +5,81 @@ group. Currently the folllowing bundles are included:
...
 * *ac.soton.rodinp.target*: Bundle including the released features
   including their source and tests (SDK) from our group. Additionally,
   this bundle also include the SDK of Eclipse features that our
   features depend on. This bundle is intended for developers and to be
   used as the target platform. Currently, the content of the bundle is
   as follows
-  - Rodin Product 3.5.0.202009111309-74e0e4188
-  - CamilleX SDK 2.0.0.release
+  - Rodin Product 3.6.0.202105121522-77c344946
+  - Rodin Handboock 2.5.0.201606291411
```
Comment (cfsnook) [inline README.md]: Handbook

File: README.md [hunk 4]
```diff
@@ -5,71 +5,81 @@ group. Currently the folllowing bundles are included:
...
+  - RMF ProR 0.13.0.201509161042
 
 * *ac.soton.rodinp.baseline*: Bundle including the released features
   including their source and tests (SDK) from our group. This bundle
   is intended for developers and to be used as the API baseline.
   Currently, the content of the bundle is as follows
-  - Rodin Product 3.5.0.202009111309-74e0e4188
-  - CamilleX SDK 2.0.0.release
+  - Rodin Product 3.6.0.202105121522-77c344946
+  - Rodin Handboock 2.5.0.201606291411
```
Comment (cfsnook) [inline README.md]: Handbook

File: README.md [hunk 5]
```diff
@@ -5,71 +5,81 @@ group. Currently the folllowing bundles are included:
 * *ac.soton.rodinp.releases*: Bundle including the released features
   from our group. This bundle is intended for users. Currently, the
   content of the bundle is as follows
-  - Rodin Product 3.5.0.202009111309-74e0e4188
-  - CamilleX 2.0.0.release
+  - Rodin Product 3.6.0.202105121522-77c344946
+  - Rodin Handboock 2.5.0.201606291411
```
Comment (cfsnook) [inline README.md]: Handbook

**Review by cfsnook — CHANGES_REQUESTED** [review]
readm.md has a few minor typos

**Comment by tshoang** [comment]
@cfsnook Thanks. I've fixed the typos.

**Review by cfsnook — APPROVED** [review]
great, it is perfect now

**Pull request merged**